client/src/inside/components/page/ehr-helper.js

```diff
@@ -509,6 +509,26 @@ export default class EhrHelpV2 {
     router.go(0)
   }
 
+  async resetFormData (childrenKeys) {
+    const { pageKey } = this
+    
+    const ehrSeed = StoreHelper.getSeedEhrData()
+    const asLoadedPageData = this.getAsLoadedPageData()
+    childrenKeys.map(ck => {
+      const asSeed = ehrSeed[ck] ? ehrSeed[ck] : ''
+      asLoadedPageData[ck] = asSeed
+    })
```

Comment:
There is a more direct and future proof approach. See clearTable at line 379.  In fact I think we can refactor clearTable to be "clearPageElement" and pass in either the table key or the form key.

Reply:
I like this approach! I will be working on it tomorrow! Thank you!